### PR TITLE
Rm paid tasks

### PR DESCRIPTION
**Pull request by phlax**
As this functionality is unsupported, and the code is untested and has a history of security issues this PR removes it

### CODE REVIEW

File: pootle/static/js/admin/components/User/UserForm.js [hunk 1]
```diff
@@ -159,8 +159,6 @@ const UserForm = React.createClass({
           <ul className="action-links">
             <li><a href={model.getProfileUrl()}>{gettext('Public Profile')}</a></li>
             <li><a href={model.getSettingsUrl()}>{gettext('Settings')}</a></li>
-            <li><a href={model.getStatsUrl()}>{gettext('Statistics')}</a></li>
```
Comment (phlax) [inline pootle/static/js/admin/components/User/UserForm.js]: this view is removed

Comment (unho) [inline pootle/static/js/admin/components/User/UserForm.js]: If removed then you might consider dropping `getStatsUrl` too.

Comment (phlax) [inline pootle/static/js/admin/components/User/UserForm.js]: yep, good point

**Comment by dwaynebailey** [comment]
lgtm, withe caveat that we'll need to get something back for 2.8.0

**Comment by unho** [comment]
I am not sure if removing all user's stats is something I really like, but won't block if that is considered to be back in a better way. lgtm once Travis passes.

**Comment by phlax** [comment]
agreed - opening a ticket...

https://github.com/translate/pootle/issues/5356

**Pull request closed without merging**